src/test/java/lotto/domain/ValidationTest.java

```diff
@@ -0,0 +1,70 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.List;
+import java.util.Set;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class ValidationTest {
+
+    @Test
+    void validLottoNoTest() {
+        int underMin = LottoNo.MIN_NUMBER - 1;
+        int min = LottoNo.MIN_NUMBER;
+        int upperMax = LottoNo.MAX_NUMBER + 1;
+        int max = LottoNo.MAX_NUMBER;
+
+        boolean underMinValidation = Validation.validateLottoNo(underMin);
+        boolean minValidation = Validation.validateLottoNo(min);
+        boolean upperMaxValidation = Validation.validateLottoNo(upperMax);
+        boolean maxValidation = Validation.validateLottoNo(max);
+
+        assertThat(underMinValidation).isFalse();
+        assertThat(minValidation).isTrue();
+        assertThat(upperMaxValidation).isFalse();
+        assertThat(maxValidation).isTrue();
+    }
+
+    @ParameterizedTest
+    @CsvSource(
+            "'1,2,3,3,5|1,2,3,4,5,5|1,2,3,4,5,6'"
+    )
+    void checkValidationLottoTicketTest(String input){
+        List<Set<Integer>> ticketsNumber = CsvParsing.convertStringToTicketsNumber(input);
+
+        boolean invalidNumberSize = Validation.validateLottoTicket(ticketsNumber.get(0));
+        boolean invalidDuplication = Validation.validateLottoTicket(ticketsNumber.get(1));
+        boolean validNumber = Validation.validateLottoTicket(ticketsNumber.get(2));
+
+        assertThat(invalidNumberSize).isFalse();
+        assertThat(invalidDuplication).isFalse();
+        assertThat(validNumber).isTrue();
+    }
+
+    @ParameterizedTest
+    @CsvSource(
+            "'1,2,3,4,5,6|1,2,3,4,5,6|1,2,3,4,5,6|1,2,3,4,5,5|1,2,3,4,5,6'," +
+                    "'3,0,46,0,7'"
+    )
```

Comment:
로또번호, 보너스번호를 테스트하고 싶으신걸까요~?
아래와 같이 좀 더 가독성있게 테스트케이스를 작성하실 수 있습니다.

```suggestion
@CsvSource(value = {
            "1,2,3,4,5,6 | 3",
            "1,2,3,4,5,6 | 0",
            "1,2,3,4,5,6 | 46",
            "1,2,3,4,5,5 | 0",
            "1,2,3,4,5,6 | 7"
    }, delimiter = '|')
```

Reply:
정말 감사합니다! 이걸 몰라서 노가다로 다 적어주고 있었네요 ㅠㅠ...